### PR TITLE
Remove Declaration of War

### PR DESCRIPTION
**Pull request by Tryded**
Removes the ability for nuclear operatives to declare war.

## About the PR

Takes away the commander's declaration of war remote. Still can get spawned by admeme.
## Why / Balance

WarOps are usually just LRP, turning into a deathmatch between the station and nuclear operatives.
## Technical details

Edited the commander's loadout to not have the Declaration of War.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by evilexecutive** [comment]
It would probably be best to only remove it from starting equipment, but not from the game's code. So as to prevent potential merge conflicts.

**Comment by Tryded** [comment]
Alright, I'll try to add back the files.

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml [hunk 1]
```diff
@@ -135,8 +135,6 @@
     pocket1: DoubleEmergencyOxygenTankFilled
     pocket2: BaseUplinkRadio40TC
     belt: ClothingBeltMilitaryWebbing
-  inhand:
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:138]: All you would need is to comment out those two lines and keep the rest.
This technically allows the least amount of conflicts and admins can still spawn them in if they want to for events.

**Comment by Tryded** [comment]
Is there an easy way to remove some of the changes?

**Comment by Tryded** [comment]
That should have done it.

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml (hunk 1 above)
Comment (Colin-Tel) [reply Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:138]: This point is still valid. Don't remove the lines, just comment them out.

**Review by Colin-Tel — CHANGES_REQUESTED** [review]

**Comment by Tryded** [comment]
I have figured out what does commenting out mean, it has been done.

**Review by Colin-Tel — DISMISSED** [review]

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml [hunk 2]
```diff
@@ -135,8 +135,8 @@
...
-    right hand: NukeOpsDeclarationOfWar
+ #inhand:
```
Comment (DebugOk) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:138]: Missing the reason behind the comment

**Review by DebugOk — CHANGES_REQUESTED** [review]
See previous comment

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml (hunk 2 above)
Comment (Colin-Tel) [reply Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:138]: Yeah, fair enough.

**Comment by Tryded** [comment]
I have finally (maybe) finished editing 2 lines of code

**Review by DebugOk — APPROVED** [review]

**Pull request merged**